components/create_post/create_post.jsx

```diff
@@ -1055,7 +1060,7 @@ export default class CreatePost extends React.Component {
                 <span
                     role='button'
                     tabIndex='0'
-                    aria-label={Utils.localizeMessage('create_post.open_emoji_picker', 'Open emoji picker')}
+                    aria-label={formatMessage({id: 'create_post.open_emoji_picker', defaultMessage: 'Open emoji picker'})}
```

Comment:
I like that you're still using the intl object in cases like this that are already deeply nested since a `FormattedMessage` would just make it even more complicated